src/js/edu-benefits/1990/config/migrations.js

```diff
@@ -0,0 +1,15 @@
+import _ from 'lodash/fp';
+
+export default [
+  // 0 -> 1, we have split the edu bundle into form-specific bundles with a new urlPrefix
+  // and this replaces the saved returnUrl if necessary
+  (savedData) => {
+    const savedReturnUrl = _.get('returnUrl', savedData);
```

Comment:
Have you tested this? I don't think `returnUrl` is in here, I think we need to change some of the common SiP code to pass through the returnUrl and use any updates to it.

Reply:
Good point, change made.